scene/gui/progress_bar.cpp

```diff
@@ -91,12 +91,31 @@ void ProgressBar::_notification(int p_what) {
 			}
 
 			if (show_percentage) {
-				String txt = itos(int(get_as_ratio() * 100));
+				double ratio = 0;
+
+				if (Math::is_equal_approx(get_max(), get_min())) {
```

Comment:
Maybe it's worth it to add a comment that it's basically copy-paste of `get_as_ratio()`, but unclamped.